src/components/text-input-email.components.js

```diff
@@ -0,0 +1,22 @@
+import $ from 'jquery';
+import TextInput from './text-input.components';
+import InvalidEmail from '../shared/erros/text-input-email.erros';
+
+export default class EmailTextInput extends TextInput {
+    constructor(stringQuery, required = true, _textValid = false, disabled = false) {
+        super(stringQuery, required, _textValid, disabled);
+        this._textInput = $(`${stringQuery} input[type="email"]`);
+    }
+
+    isValid() {
+        // Regex with a email form
+        const EmailRegex = /^([a-zA-Z0-9_.+-])+\@(([a-zA-Z0-9-])+\.)+([a-zA-Z0-9]{2,4})+$/;
```

Comment:
Here's a lint error
![image](https://user-images.githubusercontent.com/92385147/164703297-9cfe5f70-06e0-4c3f-8154-875d880f3572.png)

Reply:
I changed the regex